src/auxinfo/auxinfo_commit.rs

```diff
@@ -69,7 +69,7 @@ impl AuxInfoDecommit {
         let mut u_i = [0u8; 32];
         rng.fill_bytes(rid.as_mut_slice());
         rng.fill_bytes(u_i.as_mut_slice());
-
+        // How should the context be defined here ?
         public_keys.verify()?;
```

Comment:
@marsella this is in reference to my comment about AuxInfoPublic and the context passed to proof/verify.